api/metrics/views.py

```diff
@@ -320,43 +320,56 @@ class RecentReportList(JSONAPIBaseView):
         MetricsReportsTsvRenderer,
     )
 
-    def get(self, request, *args, report_name):
+    def get_default_search(self):
```

Comment:
this is getting beyond the scope of the ticket, but it'd be nice for `RecentReportList` to inherit `ElasticsearchListView`, now that it exists (instead of only duck-typing `get_default_search`) -- could be fuller-featured and more consistent, but might also rabbit-hole, so don't bother unless you want to